app/models/user.rb

```diff
@@ -1,10 +1,16 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
   has_many :posts, foreign_key: 'author_id'
   has_many :comments, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
 
+  has_one_attached :photo
+
   validates :name, presence: true
-  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  # validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] It looks like you have commented the validation for `posts_counter` while it is essential for the safety of data entry in your models. Also, you will need this in the following projects about model specs. **However, what I would like to mention is that It is among the best practices to remove all the unnecessary commented codes in order to keep your codebase clean and readable** 😃 👍  So if you choose to uncomment the above line, that's ok and it's the better but if you don't need it, please remove it instead of keeping it commented there 😎 I hope you get my point correctly